### DISCUSSION TITLE
500 error code in the version 6.1.13

### DISCUSSION THREAD

**Discussion by Greg-Bush** [firefly-iii/firefly-iii#8770] [Firefly III] [answered]
Hello!

I have just installed php 8.3 and latest version of the Firefly on clean ubuntu 22.04.

And there is 500 error code like this:
![Screenshot 2024-04-09 at 21 41 29](https://github.com/firefly-iii/firefly-iii/assets/44202486/af1923a3-939d-4ba9-ba79-02a7912f1e34)

Could someone suggest a way of inspection some logs in order to find the error info?
Or probably some way of starting the application in debug / development mode would be helpful.

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/how-to/general/debug/

**Reply by Greg-Bush** [reply]
The error was: "Vite manifest not found".

And I invoked
`cd /var/www/html/firefly-iii && npm i && npm run build`
to fix this error.

After that I passed login page and met there an empty page. And it was resolved by switching back from debug mode to production.

So thank you for helpful answer. I've finally updated to the latest version.
